app/views/targets/edit.haml

```diff
@@ -0,0 +1,42 @@
+= render 'targets/facebook_logout'
+#googleMap
+%script{src: 'http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js'}
+%script{src: 'https://maps.googleapis.com/maps/api/js?key=' + ENV['GOOGLE_API_KEY'] + '&callback=targetEdit'}
+= form_for target, url: {action: 'edit'}, html: {class:'rectangle-3'} do |f|
+  .rectangle-2
+    %p.create-target= t('edit_target')
+  = image_tag image_path('group.svg'),
+    class: 'group'
+  %p.page-title= t('edit_selected_target')
+  %p.specify-area-lenght= t('specify_area_lenght')
+  .length-rectangle
+    = f.select(:length,[['100 m',100],['200 m',200],['500 m',500],['1 km', 1000]],{},id: 'radius_select' ,class: 'length', disabled:true)
+  %p.target-title= t('target_title')
+  .title-rectangle
+    = f.text_field(:title, class:'title-input', value: target.title, disabled:true)
+    %br/
+  %p.select-a-topic= t('select_a_topic')
+  .topics-title-rectangle
+    %p.topics-title= t('talk_about')
+  .topics-rectangle
+    %ul.list#topic-list
+      - topics.each do |topic|
```

Comment:
you can select the target here in this loop instead of having javascript doing it